store/db/prefixdb_test.go

```
@@ -0,0 +1,49 @@
+package db_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
```

golangci-lint (1.62.2)

File is not `gci`-ed with --skip-generated -s standard -s default -s prefix(cosmossdk.io) -s prefix(github.com/cosmos/cosmos-sdk) --custom-order

(gci)